runtime-modules/proposals/engine/src/lib.rs

```diff
@@ -398,7 +439,7 @@ decl_module! {
         }
 
         /// Veto a proposal. Must be root.
```

Comment:
Parity uses this format:
```
T::WeightInfo::set_members(
				*old_count, // M
				new_members.len() as u32, // N
				T::MaxProposals::get() // P
			),
```
Can we use `T::WeightInfo` instead of `<T as Trait>::WeightInfo` as well?